### PR TITLE
added sqosmo

### PR DESCRIPTION
**Pull request by maxrobot**

### CODE REVIEW

File: osmosis/assetlist.json [hunk 1]
```diff
@@ -543,6 +543,27 @@
         }
       ]
     },
+    {
+      "description": "Margined Power Token SQOSMO",
+      "denom_units": [
```
Comment (JeremyParish69) [inline osmosis/assetlist.json:548]: please define a base unit (with exponent 0)

File: osmosis/assetlist.json [hunk 2]
```diff
@@ -543,6 +543,27 @@
...
+          "denom": "factory/osmo1g8qypve6l95xmhgc0fddaecerffymsl7kn9muw/squosmo",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/osmo1g8qypve6l95xmhgc0fddaecerffymsl7kn9muw/squosmo",
+      "name": "SQOSMO",
+      "display": "sqosmo",
+      "symbol": "SQOSMO",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/SQOSMO.svg"
```
Comment (JeremyParish69) [inline osmosis/assetlist.json:559]: please lowercase the image file name

File: osmosis/assetlist.json [hunk 3]
```diff
@@ -543,6 +543,27 @@
...
+      },
+      "images": [
+        {
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/SQOSMO.svg"
+        }
+      ]
+    },
```
Comment (JeremyParish69) [inline osmosis/assetlist.json:566]: Please move to the bottom of the list

Comment (JeremyParish69) [inline osmosis/assetlist.json:566]: Please define a trace to OSMO

Comment (sunnya97) [reply osmosis/assetlist.json]: I'm not sure its really a true trace to OSMO.  It's more of a wacky derivative on top

Comment (JeremyParish69) [reply osmosis/assetlist.json]: I think we'll skip defining the trace for now but I want to add a new trace type: 'derivative' allowing for multiple origin assets, accommodiating for classic leverage (e.g., 3x); this 'power leveraged'; and also index funds, aggregating from multiple tokens.

File: osmosis/assetlist.json [hunk 4]
```diff
@@ -648,6 +648,31 @@
...
+        },
+        {
+          "denom": "sqosmo",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/osmo1g8qypve6l95xmhgc0fddaecerffymsl7kn9muw/squosmo",
+      "name": "OSMO Squared",
+      "display": "sqosmo",
+      "symbol": "SQOSMO",
```
Comment (JeremyParish69) [inline osmosis/assetlist.json:667]: Please update symbol to sqOSMO

**Pull request merged**